fabric_plugin/tests/blueprint/scripts/mkdir.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
```

Comment:
why not add a test to the existing script.sh file and be consistent with other tests?